package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pick-your-movie",
+  "version": "1.0.0",
+  "description": "A web application to search for movies, with an autocomplete feature.",
+  "main": "src/sever.js",
```

Comment:
this should default to `index.js`

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#main

Reply:
Will it consider the public folder as my root? because I served the static content in my server from it, so also there is no need to put src/, right?

Reply:
it has nothing to do with how you server your files. It has something to do with exporting and requiring your modules, for example `require('/users')` this will require the main file specified in your package.json from that directory which is `index.js`. 

And you are right, for instance if we run this command `node src/` this will run the index.js file inside the src directory